transformer/task.py

```diff
@@ -43,7 +45,7 @@ def from_request(cls, r: Request) -> "LocustRequest":
     def as_locust_action(self) -> str:
         args = {
             "url": self.url,
-            "name": self.url,
+            "name": self.name if self.name else self.url,
```

Comment:
```suggestion
            "name": self.name or self.url,
```
Minor simplification.